source/op/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ if (BUILD_CPP_IF)
   target_link_libraries (${LIB_DEEPMD_OP} PRIVATE ${LIB_DEEPMD})
   target_include_directories(${LIB_DEEPMD_OP} PUBLIC ${TensorFlow_INCLUDE_DIRS})
   set_target_properties(${LIB_DEEPMD_OP} PROPERTIES INSTALL_RPATH "$ORIGIN;${TensorFlow_LIBRARY_PATH}")
+  if (USE_TF_PYTHON_LIBS)
+    set_target_properties(${LIB_DEEPMD_OP} PROPERTIES COMPILE_FLAGS -D_GLIBCXX_USE_CXX11_ABI=${OP_CXX_ABI})
```

Comment:
This is not included in api_cc/tests/CMakeList.txt

Reply:
I use the code of the devel branch. I found that the reason was the lack of `libdeepmd_op_cuda.so`. After adding a link to `libdeepmd_op_cuda.so` in `libdeepmd_op.so`, the problem was solved. Is this reasonable?

Reply:
I think it should link to libdeepmd.so

Reply:
Could you provide the entire log? The error is more like loading a previous version of library

Reply:
If I use the CPU custom op, everything works well:
```
cmake -DUSE_CUDA_TOOLKIT=false -DTENSORFLOW_ROOT=$tensorflow_root -DCMAKE_INSTALL_PREFIX=$deepmd_root .
```
The problem seems to be a link error of the custom op's GPU implementation library.

Reply:
I'm not testing with this PR, I'm using the latest devel code.

Reply:
[log.txt](https://github.com/deepmodeling/deepmd-kit/files/8997202/log.txt)

Reply:
... I think you are right. I suggest using `add_subdirectory` as we have good Cmake files outside the tests